packages/iris-grid/src/IrisGridPartitionSelector.tsx

```diff
@@ -303,15 +303,16 @@ class IrisGridPartitionSelector extends Component<
     }
     const partitionSelectors = model.partitionColumns.map((column, index) => (
       <div key={`selector-${column.name}`} className="column-selector">
-        <div className="column-name">{column.name}</div>
         {partitionTables?.[index] && (
           <Picker
+            label={column.name}
             table={partitionTables[index]}
             direction="bottom"
             shouldFlip={false}
             keyColumn={partitionTables[index].columns[index].name}
             placeholder={'Loading...' as string}
             labelColumn={partitionTables[index].columns[index].name}
+            labelPosition="side"
```

Comment:
I would've probably grouped the `label` and `labelPosition` props together (since they're both labelling the component), but :shrug:

Reply:
My bad, I thought I should have `labelColumn` and `labelPosition` together, but what you have mentioned makes sense, since the `labelColumn` is referring to something else. Should be good now